app/js/App.js

```diff
@@ -20,11 +20,11 @@ class App {
    * Start application by creating a page
    */
 
-  constructor() {
+  constructor(settings) {
     mkdirp(path.resolve(__dirname, '..', 'tmp'))
     this.pages = []
     this.current_page = 0
-    this.theme = settings.get('theme')
```

Comment:
should be something like:
```
this.settings = settings
this.setTheme(settings.get('theme'))
```
No need to set `theme` on the App object, since it always comes/sets from `Settings`. Then below, we can remove `this.theme = theme` and just use `this.settings`.